### PR TITLE
Fix syntax, set correct service name, add consistent PATHs

### PR DESCRIPTION
**Pull request by anadahz**

### CODE REVIEW

**Comment by hellais** [comment]
Didn't see this PR before writing https://github.com/TheTorProject/lepidopter/pull/74#issuecomment-245232682. Ignore the stuff about the paths in that case there.

**Comment by hellais** [comment]
I would rather there weren't two layers of nesting in the file paths. We should write the files to either `/opt/ooni/lepidopter-update/updater.py` or `/opt/ooni/updater/updater.py`, but not inside of `/opt/ooni/lepidopter-update/updater/updater.py`.

File: updater/updater.py [hunk 1]
```diff
@@ -30,14 +30,16 @@
...
 Type=simple
-ExecStart={0} --log-file /var/log/lepidopter-update.log update --watch
+Environment="UPDATER=/opt/ooni/lepidopter-update/updater/updater.py"
+ExecStart=$UPDATER --log-file /var/log/ooni/lepidopter-update.log update --watch
 TimeoutStartSec=300
 Restart=on-failure
+
 [Install]
 WantedBy=multi-user.target
 """.format(SCRIPT_INSTALL_PATH)
```
Comment (hellais) [inline updater/updater.py:40]: If you remove the `{0}` you should also remove the `.format()` call at the end. I would suggest you keep it as that performs substitution of the variable `{0}` with the actual path to the script

Comment (anadahz) [reply updater/updater.py:40]: Missed that, thanks for pointing out @hellais.

**Comment by anadahz** [comment]
I prefer to use static variables in systemd unit files as it will assure that the PATH/filename is the one intended whereas a variable substitution may give other results.

**Pull request merged**